### PR TITLE
[-] Lumber - Remove no-db option

### PR DESCRIPTION
**Pull request by DrRaider**

### CODE REVIEW

**Comment by DrRaider** [comment]
[Lumber - --no-db option does not work anymore and should be dropped](https://trello.com/c/zrJrefKQ/3939-lumber-no-db-option-does-not-work-anymore-and-should-be-dropped)

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,9 @@
 ### Added
 - Command Generate - Generate MongoDB HasMany.
 
+### Changed
+- Technical - Removed no-db option
```
Comment (arnaudbesnier) [inline CHANGELOG.md]: This is not a "technical" topic.
It is a (broken) feature removal
```suggestion
- Command Generate - Remove the broken `no-db` option.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,9 @@
 ### Added
 - Command Generate - Generate MongoDB HasMany.
 
+### Changed
```
Comment (arnaudbesnier) [inline CHANGELOG.md]: [Conventions] There is already a `Changed` section.

File: lumber-generate.js [hunk 3]
```diff
@@ -36,15 +35,13 @@ program
 
   let schema = {};
 
-  if (program.db) {
```
Comment (arnaudbesnier) [inline lumber-generate.js]: Is `db: program.db,` still useful in the General Prompter (line 15)?

Comment (DrRaider) [reply lumber-generate.js]: Missed it, thanks :)

**Pull request merged**